docs/labwc-config.5.scd

```diff
@@ -330,6 +330,50 @@ The rest of this man page describes configuration options.
 	any motion events while a keyboard is typing, and for a short while
 	after as well.
 
+## WINDOW RULES
+
+Two types of window rules are supported, actions and properties. They are
+defined as shown below. Property values can be *yes*, *no* or *unset*.
+
+```
+<windowRules>
+
+  <!-- Action -->
+  <windowRule identifier="" event="">
+    <action name="">
+  </windowRules>
+
+  <!-- Property -->
+  <windowRule identifier="">
+    <serverDecoration></serverDecoration>
+  </windowRules>
```

Comment:
`s/windowRules/windowRule/` for the closing ones.

Even though `magic malm parser` handles both, I'd suggest using the compact syntax in all examples:
```xml
  <!-- Property -->
  <windowRule identifier="some.application" serverDecorations="yes" />
```

Reply:
Thanks. Done.

Reply:
Lost a commit? :D

Reply:
Done now. Had done it in the rc.xml.all file but not the man page :smile: